felix/calc/l3_route_resolver.go

```diff
@@ -422,13 +451,17 @@ func (c *L3RouteResolver) onNodeUpdate(nodeName string, newNodeInfo *l3rrNodeInf
 
 	if nodeName == c.myNodeName {
 		// Check if our CIDR has changed and if so recalculate the "same subnet" tracking.
-		var myNewCIDR ip.V4CIDR
-		var myNewCIDRKnown bool
+		var myNewV4CIDR ip.V4CIDR
+		var myNewV4CIDRKnown bool
+		var myNewV6CIDR ip.V6CIDR
+		var myNewV6CIDRKnown bool
 		if newNodeInfo != nil {
-			myNewCIDR = newNodeInfo.CIDR
-			myNewCIDRKnown = true
+			myNewV4CIDR = newNodeInfo.V4CIDR
+			myNewV4CIDRKnown = true
```

Comment:
Very nitty, but since "myNewV6CIDRKnown" and "myNewV4CIDRKnown" are both always set to true in the same place, we can probably use a single bool here.

Reply:
Good point, thanks for catching this!